project/src/main/webapp/businessAccount.html

```diff
@@ -0,0 +1,101 @@
+          <button id="logout" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent">
+            Log out
+          </button>
+          <!-- TODO: Have a login and sign up servlet -->
+          <div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect">
+            <div class="mdl-tabs__tab-bar">
+                <a href="#businessTools" class="mdl-tabs__tab is-active">Tools</a>
+                <a href="#businessInfo" class="mdl-tabs__tab">Business Info</a>
+            </div>
+            <div class="mdl-tabs__panel is-active" id="businessTools"><br/>
+              <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--colored">
+                View my products
+              </button>
+              <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--colored">
+                Create a product set
+              </button>
+              <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--colored">
+                Add a product
+              </button>
+              <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect">
+                Delete Product/set
+              </button>
+            </div>
+            <div class="mdl-tabs__panel" id="businessInfo"><br/>
+              <form action="" method="POST" >
+                <p>Business name</p> 
+                <div class="mdl-textfield mdl-js-textfield">
+                  <input class="mdl-textfield__input" type="text" id="nickname" name="nickname">
+                  <label class="mdl-textfield__label" for="nickname">JohnDoe42</label>
+                </div><br/><br/>
+                <p>Business Address</p>
+                <div class="mdl-textfield mdl-js-textfield">
+                  <input class="mdl-textfield__input" type="text" id="street" name="street">
+                  <label class="mdl-textfield__label" for="street">Street</label>
+                </div>
+                <div class="mdl-textfield mdl-js-textfield">
+                  <input class="mdl-textfield__input" type="text" id="city" name="city">
+                  <label class="mdl-textfield__label" for="city">City</label>
+                </div><br/>
+                <div class="mdl-textfield mdl-js-textfield">
+                  <input class="mdl-textfield__input" type="text" id="state" name="state">
+                  <label class="mdl-textfield__label" for="state">State</label>
+                </div>
+                <div class="mdl-textfield mdl-js-textfield">
+                  <input class="mdl-textfield__input" type="text" id="zipCode" name="zipCode">
+                  <label class="mdl-textfield__label" for="zipCode">Zip Code</label>
+                </div>
+                <br/><br/>
+                <button type="submit" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent">
```

Comment:
Does something else need to be done to link this to the business account servlet? (I might have just missed it somewhere, or maybe you haven't gotten to it yet).

Reply:
I currently do not have a business account servlet. I'm not sure if I need one. This page is loaded with information from the account servlet, which as of now is all I need.